.gitignore

```diff
@@ -0,0 +1,4 @@
+test.md
+node_modules
+node_modules
```

Comment:
- [Optional] Kindly, clean unnecessary `node_modules` duplication to make your code clean 👍 👍